content/3_Silver/Flood_Fill.mdx

```diff
@@ -739,3 +739,66 @@ print(room_num)
 ## Problems
 
 <Problems problems="general" hideSuggestProblemButton />
+
+## Quiz
+
+<Quiz>
+	<Quiz.Question>
+		What is the minimum number of starting nodes for flood fills we would need to visit every node in an undirected graph with four components?
+		<Quiz.Answer>
+			1
+			<Quiz.Explanation>
+				Incorrect. Each DFS can travel one connected component, so a graph with four connected components would require $\mathbf{4}$ DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer>
+			2
+			<Quiz.Explanation>
+				Incorrect. Each DFS can travel one connected component, so a graph with four connected components would require **4** DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer>
+			3
+			<Quiz.Explanation>
+				Incorrect. Each DFS can travel one connected component, so a graph with four connected components would require **4** DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+		<Quiz.Answer correct>
+			4
+			<Quiz.Explanation>
+				Correct! Each DFS can travel one connected component, so a graph with four connected components would require **4** DFS runs.
+			</Quiz.Explanation>
+		</Quiz.Answer>
+	</Quiz.Question>
+
+	<Quiz.Question>
+		Which of the following is not information we could get from the results of an unmodified floodfill algorithm?
+		<Quiz.Answer>
+			The number of connected components of a graph
+			<Quiz.Explanation>
+				Incorrect. We find the number of connected components by tracking the number of times DFS is executed.
```

Comment:
technically it's executed a lot of times, you probably mean the amount of initial calls